doc/cug.tex

```diff
@@ -4911,8 +4911,12 @@ \subsubsection{Custom Jinja2 Filters}
 
 In the filter function argument list, the first argument is the variable
 value to be ``filtered'', and subsequent arguments can be whatever is
-needed. Currently there is one custom filter called ``pad'' in the
-central cylc Jinja2 filter directory, for padding string values to some
+needed. Currently there are two custom filters:
+
+
+\paragraph{pad}
+
+The ``pad'' filter is for padding string values to some
 constant length with a fill character - useful for generating task names
 and related values in ensemble suites:
```

Comment:
Incidentally, I have a feeling Jinja2 has a built in way of padding, so this is unnecessary. However, this isn't new so not an issue for this PR.